scripts/build-modm.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
We already have a complete setup with make, re-implementing it in a shell script is just duplicating the functionality.

Reply:
Sounds good, this was in place in order to build the binaries for modm at a specific version directly from source. We'll leave this file as part of the PR, and it can get removed once we have a understanding of how to run an image with all processes from a locally built source from make.